ext/MeasurementsSymbolicsExt.jl

```diff
@@ -0,0 +1,35 @@
+### MeasurementsSymbolicsExt.jl
+#
+# Copyright (C) 2019 Mosè Giordano.
```

Comment:
https://github.com/JuliaPhysics/Measurements.jl/pull/145#discussion_r1199801058

Reply:
(The reason why we (as in, people in general, not just me and @longemen3000)
keep parroting this line, is because different projects have different thoughts
on what should be done about copyright. When almost all code is copyrighted by
the same author, it may be assumed that it is the preferred outcome,
in the sense of copyright assignment).